dockerfiles/README.md

```diff
@@ -0,0 +1,116 @@
+if you would like to use Docker Compose.
+
+To stop the container:
+
+```bash
+docker stop skosmos-web
+```
+
+The container created is based on the project
+[Install Tutorial](https://github.com/NatLibFi/Skosmos/wiki/InstallTutorial).
+So it will create a container with Ubuntu, Apache2, PHP, composer, and a version
+of Skosmos.
+
+The Apache virtual host configuration is located at `config/000-default.conf`. And
+the configuration file used for Skosmos is at `config/config.ttl`. Customize these
+two files as necessary.
+
+**NOTE**: If you would like to start a Fuseki container to test with Docker only,
+without Docker Compose, you can try the following command before loading your
+vocabulary data. It starts a container in the same way our other example with
+the `docker-compose` command.
+
+```bash
+docker run --name fuseki -ti --rm \
+  --env "ADMIN_PASSWORD=admin" --env "JVM_ARGS=-Xmx2g" \
+  -p 3030:3030 \
+  --mount type=bind,src=$(pwd)/config/skosmos.ttl,dst=/fuseki/configuration/skosmos.ttl \
+  stain/jena-fuseki
+```
+
+## Running with docker-compose
+
+The `docker-compose` provided configuration will prepare three containers.
+The first one called `skosmos-fuseki`, which uses the `stain/jena-fuseki`
+image for Jena, and starts a container with 2 GB of memory and `admin` as
+the user and password. The `docker-compose` service name of this container
+is `fuseki`.
+
+The second container is the `fuseki-cache`, a Varnish Cache container. It sits
+between the `skosmos-fuseki` and the `skosmos-web` (more on this below). The
+Varnish Cache container is pre-configured to intercept queries to `fuseki:3030`
+keeping the results `gzipped` in the cache for one week.
+
+The last container created is `skosmos-web`, using the same image mentioned
+in the [previous section](#running-with-docker). The only difference being
+that we bind a new Skosmos configuration `config/config-docker-compose.ttl`
+on `/var/www/html/config.ttl`.
+
+This `config-docker-compose.ttl` file uses `http://fuseki-cache:80/skosmos/sparql`
+as `skosmos:sparqlEndpoint`, forcing `skosmos-web` to go through the `fuseki-cache`
+for a better performance. You can customize this example setup to start Skosmos
+pointing to any other existing Apache Jena server, preferably with the Jena Text
+extension.
+
+**NOTE**: `fuseki:3030` and `fuseki-cache:80` are from the internal Docker network.
+To the host machine Docker Compose is exposing these values as `localhost:3030`
+and `localhost:9031` respectively.
+
+To create the containers in this example setup, you can use this command:
+
+```bash
+docker-compose up -d
+```
+
+Now Skosmos should be available at `http://localhost:9090/` from your
+host. See the [section below](#loading-vocabulary-data) to load vocabulary data.
+
+To stop:
+
+```bash
+docker-compose down
+```
+
+## Loading vocabulary data
+
+After you have your container running, with either Docker or `docker-compose`,
```

Comment:
In the docker-compose case, the ports need to be changed to 9030 in the commands below, otherwise they won't work (or will access the wrong Fuseki)